packages/api/package.json

```diff
@@ -84,6 +84,7 @@
     "isomorphic-webcrypto": "^2.3.8",
     "js-yaml": "^3.13.1",
     "jsonwebtoken": "^8.5.1",
+    "lodash": "^4.17.21",
```

Comment:
did the lock file not update because of this?

Reply:
It did not because `www` already uses `lodash`! So I believe yarn just added a dependency on the same version